request_a_govuk_domain/request/migrations/0024_application_approval_or_rejection_comment_and_more.py

```diff
@@ -0,0 +1,142 @@
+                        "Approved - Reviewer sent back to change domain name before 2i",
+                    ),
+                    (
+                        "approved_reviewer_check_registrant_before_2i",
+                        "Approved - Reviewer sent back to check Registrant before 2i",
+                    ),
+                    (
+                        "approved_reviewer_check_acronym_before_2i",
+                        "Approved - Reviewer sent back to check acronym before 2i",
+                    ),
+                    (
+                        "approved_reviewer_check_registry_details_before_2i",
+                        "Approved - Reviewer sent back to check Registry published details before 2i",
+                    ),
+                    (
+                        "approved_reviewer_change_name_before_2i",
+                        "Approved - Reviewer sent back to change name before 2i",
+                    ),
+                    ("rejected_with_nac", "Rejected with NAC"),
+                    ("rejected_duplicate_application", "Rejected - duplicate application"),
+                    ("rejected_registrar_error", "Rejected - Registrar error"),
+                ],
+                default="new",
+                max_length=100,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="historicalapplication",
+            name="status",
+            field=models.CharField(
+                choices=[
+                    ("approved", "Approved"),
+                    ("rejected", "Rejected"),
+                    ("in_progress", "In Progress"),
+                    ("ready_2i", "Ready for 2i"),
+                    ("more_information", "More Information"),
+                    ("with_nac", "Currently with NAC"),
+                    ("new", "New"),
+                    ("duplicate_application", "Duplicate application"),
+                    ("archive", "Archive"),
+                    ("failed_confirmation_email", "Failed Confirmation Email"),
+                    ("failed_decision_email", "Failed Decision Email"),
+                    ("approved_2i_check_acronym", "Approved - 2i sent back to check acronym"),
+                    ("approved_2i_check_registrant", "Approved - 2i sent back to check Registrant"),
+                    ("approved_2i_change_domain_name", "Approved - 2i sent back to change domain name"),
+                    ("approved_2i_get_permission_letter", "Approved - 2i sent back to get permission letter"),
+                    (
+                        "approved_2i_check_registry_details",
+                        "Approved - 2i sent back to check Registry published details",
+                    ),
+                    ("approved_parked", "Approved - Parked"),
+                    ("approved_went_through_nac", "Approved - Went through NAC"),
+                    ("approved_khidr_nigel_emma_discussion", "Approved - Khidr, Nigel, Emma discussion"),
```

Comment:
Please never use person names, especially in public code. Use something like "Approved - Domains team discussion" or something.